.github/workflows/docs.yaml

```diff
@@ -26,13 +27,16 @@ jobs:
 
     - name: Replace old documentation with new
       run: |
-        [ -d docs ] || mkdir docs
-        [ -d docs/html ] && rm -r docs/html
+        (cd docs && git rm -r "*")
+        cp repo/doc/documentation.html docs/index.html
```

Comment:
I'm curious about why add this index.html that didn't exist previously

Reply:
You can see the impact by comparing https://nasa.github.io/astrobee/ to https://trey0.github.io/astrobee/ . The former comes from GitHub not finding an `index.html` file and falling back to rendering the `README.md` file. That's not awful, but it seemed to me that any users who might land on the short URL would probably prefer to get redirected straight to the docs.

And actually, how I came to add it was a suggestion in the documentation for the GitHub Pages system that it uses the existence of an `index.html` file to infer what folders within the Pages branch should be offered as choices to become the documentation root. I'm not sure this is key, but it might help future developers who are activating the Pages feature for their fork. Mine was previously activated but in a weird broken state that took a while to figure out.